composer.json

```diff
@@ -30,7 +30,7 @@
     "cultuurnet/silex-amqp": "~0.1",
     "cultuurnet/silex-service-provider-jwt": "~0.1",
     "cultuurnet/udb2-domain-events": "~0.1",
-    "cultuurnet/udb3": "dev-master#7b233a909118737b2cafec49fed1adde3b8d5ba0",
+    "cultuurnet/udb3": "~0.1",
```

Comment:
I was able to remove the fixed commit hash here this time. Composer didn't want to update to this commit in the past, but this time it updated to the latest commit without problem by using `~0.1`. So moving forward this is better because then we can just run `composer up cultuurnet/udb3` without having to change the commit hash.